<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChineseTypewriter</class>
 <widget class="QMainWindow" name="ChineseTypewriter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chinese Typewriter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_10">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_11">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_13">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_14">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_16">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_17">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_18">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_19">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_20">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_21">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_22">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_23">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_24">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_25">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_26">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_27">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="selectorButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>410</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>15</y>
      <width>1192</width>
      <height>96</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>1192</width>
      <height>256</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionCloseFile"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNewFile">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>NewFile</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>SaveFile</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>CloseFile</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
